packages/fxa-auth-server/test/local/redis.js

```diff
@@ -50,7 +50,7 @@ const sessionToken = {
 describe('Redis', () => {
   after(async () => {
     await redis.del(uid);
-    await redis.close();
+    redis.close();
```

Comment:
Do keeping this test changes break anything?

Reply:
If I recall, it _might_ have contribute to some flakiness. I think the effect of this change was small though.